packages/atlas/src/components/_auth/LogInModal/LogInModal.styles.ts

```diff
@@ -1,9 +1,14 @@
 import styled from '@emotion/styled'
 
+import { TextButton } from '@/components/_buttons/Button'
 import { sizes } from '@/styles'
 
 export const Container = styled.div`
   display: flex;
   flex-direction: column;
   gap: ${sizes(6)};
 `
+
+export const ForgotPasswordText = styled(TextButton)`
+  margin: 0 0 0 auto;
```

Comment:
`margin-left: auto`  would be much easier to read

Reply:
I needed to reset other margins to 0, that's the reason

Reply:
so you're saying that TextButton has some default margins applied?